content/features/3-pooling.mdx

```diff
@@ -65,19 +65,20 @@ pool
         client.release()
         console.log(err.stack)
       })
-  })(
-    // async/await - check out a client
-    async () => {
-      const client = await pool.connect()
-      try {
-        const res = await client.query('SELECT * FROM users WHERE id = $1', [1])
-        console.log(res.rows[0])
-      } finally {
-        client.release()
-      }
-    }
-  )()
-  .catch(e => console.log(e.stack))
+  })
+
+// async/await - check out a client
+(async () => {
```

Comment:
The original reason reformatting combined them is that this is an ASI reliance mistake when it’s one script.

```suggestion
;(async () => {
```

Reply:
Formatted like that, really: https://github.com/brianc/node-postgres-docs/commit/f38a3ab17959e9f5cb22d04d34a5ae8f75a839fd

Reply:
thanks, fixed

Reply:
It goes before the `(`, rather than on the preceding line.

Reply:
oh, that's a weird semicolon placement. I noticed you don't like semicolons much :)  Will change.

Reply:
Yep, it’s weird! I do like semicolons, but the pg style is to omit them.